### PR TITLE
Added function to get Pod status

### LINKED ISSUE #59: Add function to check pod status

### PR DESCRIPTION
**Pull request by yathamravali**
Signed-off-by: Ravali Yatham <yatraval@in.ibm.com>

Closes #59

### CODE REVIEW

File: src/main/java/com/kruize/metrics/PodMetrics.java [hunk 1]
```diff
@@ -111,8 +111,8 @@ public double getCurrentCpuLimit()
         return currentRecommendations.getCpuLimit();
     }
 
-    public double getCurrentRssLimit()
-    {
-        return currentRecommendations.getRssLimit();
-    }
+    public double getCurrentRssLimit() { return currentRecommendations.getRssLimit(); }
+
```
Comment (dinogun) [inline src/main/java/com/kruize/metrics/PodMetrics.java]: Can you please delete the additional lines here.

File: src/main/java/com/kruize/metrics/ContainerMetrics.java [hunk 2]
```diff
@@ -115,8 +115,6 @@ public double getCurrentCpuLimit()
     }
 
     @Override
-    public double getCurrentRssLimit()
-    {
-        return currentRecommendations.getRssLimit();
+    public double getCurrentRssLimit() { return currentRecommendations.getRssLimit();
```
Comment (dinogun) [inline src/main/java/com/kruize/metrics/ContainerMetrics.java]: If you are moving to a single line, please move the ending bracket as well.

Comment (yathamravali) [reply src/main/java/com/kruize/metrics/ContainerMetrics.java]: Okay.

File: src/main/java/com/kruize/metrics/PodMetrics.java (hunk 1 above)
Comment (yathamravali) [reply src/main/java/com/kruize/metrics/PodMetrics.java]: Done, Deleted as requested. Thanks!!

**Comment by ShishirH** [comment]
Thank you, and congrats on your first contribution to the Kruize project @yathamravali !

**Pull request merged**